contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
+
+We welcome everyone to contribute what they can. Whether you are brand new, just want to contribute a little bit, or want to contribute a lot there is probably something you can help out with.
+
+## Where to get help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it's designed for those who are new so please don't let length intimidate you.
+
+## Code of Conduct
+Please be nice when interacting with others and follow our [code of conduct](<link to code of conduct>).
+
+## How to contribute
+If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
+
+### Contributing by adding an issue
+If you have found a bug and would like to report it or if you have a feature that you feel we should add, then we'd love it if you opened an issue! ❤️ Before you do, please search the other issues to avoid creating a duplicate issue. 
+To submit a new issue just hit the issue button and a choice between two templates should appear. Then, follow along with the template you chose. If you don't know how to fill in all parts of the template go ahead and skip those parts. You can edit the issue later.
+
+### Contributing through code
+In order to contribute through code follow the steps below. Note that you don't need to be the best programmer to contribute. Our discord is open for questions
+
+ 1. **Pick a feature** you would like to add or a bug you would like to fix
+	 - If you wish to contribute but what you want to fix/add is not already covered in an existing issue, please open a new issue
+ 2. **Discuss** the issue with the rest of the community
```

Comment:
Yeah, the question here is a discussion on the issue or via a GH discussion: https://docs.github.com/en/discussions. I give @bgins the power to make the best partition for our cases :).

Reply:
Yep, discussions vs issues here: https://github.com/fission-codes/rust-template/pull/4/files#r1015784880

Generally, I'd say the use cases for discussions are less about contributing through code. So this section seems fine as is -- we still want to discuss in issues.